lib/server/utils.js

```diff
@@ -54,9 +56,18 @@ function minifyCss(cssContent) {
     .then(result => result.css);
 }
 
+function autoPrefixCss(cssContent) {
+  return postcss([autoprefixer])
+    .process(cssContent, {
+      from: undefined,
```

Comment:
is this `from: undefined,` required ?

Reply:
Yes. The `from` option is for defining the source file for source maps, but the current implementation rewrites the source file, so a source map is not possible. 

Without the `from: undefined`, an error is logged to the console asking to include the `from: undefined` option.